cclib/parser/molcasparser.py

```diff
@@ -434,14 +434,14 @@ def extract(self, inputfile, line):
                     enthalpy_values.append(float(next(inputfile).split()[1]))
                     free_energy_values.append(float(next(inputfile).split()[3]))
 
-                line=next(inputfile)
+                line = next(inputfile)
             # When calculations for more than one temperature value are
             # performed, the values corresponding to room temperature (298.15 K)
             # are returned and if no calculations are performed for 298.15 K, then
             # the values corresponding last temperature value are returned.
-            index  = -1
+            index = -1
             if 298.15 in temperature_values:
-            	index = temperature_values.index(298.15)
+                index = temperature_values.index(298.15)
```

Comment:
Is `298.15` always guaranteed to be one of the temperatures? Perhaps `index` should default to 0 instead of -1?

Reply:
I think we discussed this back when we were pushing it earlier. 298.15 is taken because that is the temperature we prefer for all the values. We also discussed about changing the current structure for parsing Thermochemistry values and you guys set it as a goal for version 2? (Not sure about this). -1 was taken just to get the last parsed value.

Reply:
Oh, duh. My brain was in Java mode and didn't like the `-1` index. Of course, that's the better choice here.